WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -497,7 +497,7 @@ private async Task CreateRoundsAsync(CancellationToken cancellationToken)
 				x.Phase == Phase.InputRegistration
 				&& x is not BlameRound
 				&& !x.IsInputRegistrationEnded(x.Parameters.MaxInputCountByRound)
-				&& x.InputCount >= roundDestroyerInputCount).ToArray())
+				&& x.InputCount >= Math.Min(0.9 * x.Parameters.MaxInputCountByRound, roundDestroyerInputCount)).ToArray())
```

Comment:
`roundDestroyerInputCount` has no use after this change and at LOC540 we will log false data.

How about this?

```
var roundDestroyerInputCount = 0.9 * x.Parameters.MaxInputCountByRound
```

Then, we don't complicate further complicate this `Where` predicate.

Reply:
hehe, `Config.MaxInputCountByRound` is better.

Edited my prev message.

Reply:
I tried to fix this while respecting that we want use the `Round Parameter` and not the `Config`
https://github.com/zkSNACKs/WalletWasabi/pull/12591